recipes-connectivity/iot-companion/iot-companion_1.1.bb

```diff
@@ -0,0 +1,16 @@
+SUMMARY = "IoT Companion - Commandline helper application"
+
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+
+SRC_URI = "\
+    file://iot-companion.sh \
+"
+
+RDEPENDS_${PN} += " jq"
+
+do_install () {
+    install -d ${D}${bindir}/
+    install -m 0755 ${WORKDIR}/iot-companion.sh ${D}${bindir}/iot
```

Comment:
Installing the script as `iot` does not seem like a good name for me. `iot` is way too generic. Please use the programs name `iot-companion` or something more specific.

Reply:
`iot-companion` is used as the program name. And a symlink is also added to `iot` with `ln -sf iot-companion ${D}/usr/bin/iot`